_common/KeymanHosts.php

```diff
@@ -107,12 +109,29 @@ function __construct() {
         $this->keyman_com = "https://keyman-staging.com";
         $this->keymanweb_com = "https://keymanweb.com";
         $this->r_keymanweb_com = "https://r.keymanweb.com";
+      } else if($this->tier == KeymanHosts::TIER_TEST) {
+        $this->s_keyman_com = "https://s.keyman.com";
+        $this->api_keyman_com = "http://host.docker.internal:8058"; // Unique for api.keyman.com
```

Comment:
> This smells like we are no longer testing against the static test data. I wonder if the changes to hosts mean that the env is no longer TIER_TEST or something like that?

Ah. I see the original TIER_TEST used `https://api.keyman-staging.com` and not the actual repo?
https://github.com/keymanapp/api.keyman.com/blob/60a7ab4f0b4da5d341ac90606f71d90e2b58db64/_common/KeymanHosts.php#L100-L104

Reply:
> Ah. I see the original TIER_TEST used `https://api.keyman-staging.com` and not the actual repo?

TIER_TEST uses local test data, so we can ensure that we get consistent results.

Reply:
I'm still confused.

> $this->api_keyman_com = "https://api.keyman-staging.com";

Doesn't this mean the live staging branch ci would check api.keyman-staging.com for its tests?
My change was setting to `http://host.docker.internal:8058`, which I thought would be local test data.

Reply:
> Seems the khmer search uses a weight 245 for sil_khmer but the expected weight 356.

This test seems to be passing now, so I think the only problem left is the test failure I highlighted earlier:

> https://github.com/keymanapp/api.keyman.com/actions/runs/6009846050/job/16300063487?pr=206#step:4:1765
> 
> ```
>    │              "sourcePath": "release/k/khmer_angkor",
>    │ -            "url": "https://keyman-staging.com/go/package/download/khmer_angkor?version=1.0.6&platform=windows&tier=alpha&update=0",
>    │ +            "url": "https://keyman.com/go/package/download/khmer_angkor?version=1.0.6&platform=windows&tier=alpha&update=0",
>    │              "version": "1.0.6"
> ```
> 
> This shows that the test is expecting keyman-staging.com (and help.keyman-staging.com) in KeymanHosts for TIER_TEST.

You wrote:
> Doesn't this mean the live staging branch ci would check api.keyman-staging.com for its tests?

Yes, using host.docker.internal is correct. But we can see:

https://github.com/keymanapp/api.keyman.com/blob/d7181388a4f4ea205c0061c41e278d168f83539f/script/windows/14.0/update/WindowsUpdateCheck.php#L190-L195

This is still picking up the wrong answer; I think we need it to be keyman-staging for TIER_TEST.